src/riakc_poolboy.app.src

```diff
@@ -2,7 +2,7 @@
 {application, riakc_poolboy,
  [
   {description, "Simple Riak PB connection pool"},
-  {vsn, "1.3.1"},
+  {vsn, "1.4.0"},
```

Comment:
Yeah, something like that. I'm just not sure to also add `riakc_poolboy`-specific increments in a non-ugly way though... `$riakc_version-$riakc_poolboy_version` comes to mind, but seems kinda long...